src/storage/src/types/sinks.proto

```diff
@@ -23,8 +23,7 @@ message ProtoComputeSinkDesc {
     mz_repr.global_id.ProtoGlobalId from = 1;
     mz_repr.relation_and_scalar.ProtoRelationDesc from_desc = 2;
     ProtoComputeSinkConnection connection = 3;
-    optional ProtoSinkEnvelope envelope = 4;
-    ProtoSinkAsOf as_of = 5;
+    ProtoSinkAsOf as_of = 4;
```

Comment:
This is a breaking change. I figured it would be fine since we are planning to make the next release breaking anyway. If anyone disagrees, we can also make this non-breaking by making tag 4 a reserved field.